src/axolotl/loaders/adapter.py

```diff
@@ -93,6 +93,13 @@ def load_lora(
     if loftq_bits:
         lora_config_kwargs["loftq_config"] = LoftQConfig(loftq_bits=loftq_bits)
         lora_config_kwargs["init_lora_weights"] = "loftq"
+
+    if cfg.adapter == "qalora":
+        if hasattr(cfg, "use_qalora") and cfg.use_qalora:
+            lora_config_kwargs["use_qalora"] = True
+        if hasattr(cfg, "qalora_group_size") and cfg.qalora_group_size:
+            lora_config_kwargs["qalora_group_size"] = cfg.qalora_group_size
```

Comment:
Do you want to raise an Error here or possibly in validation if a user does not provide this?